Client/mods/deathmatch/logic/luadefs/CLuaVehicleDefs.cpp

```diff
@@ -4340,3 +4342,19 @@ bool CLuaVehicleDefs::SpawnVehicleFlyingComponent(CClientVehicle* const vehicle,
 
     return vehicle->SpawnFlyingComponent(partNodeIndex, collisionType, removalTime.value_or(-1));
 }
+
+bool CLuaVehicleDefs::SetSmokeTrailEnabled(CClientVehicle* vehicle, bool state) noexcept
+{
+    auto model = vehicle->GetModel();
```

Comment:
We try not to use ``auto`` for simple variable types, so I think it's better to just change it to ``std::uint16_t``

Reply:
you are right I wonder why tracer suggest auto @TracerDS

Reply:
for simplicity as an example. Doesnt really matter which one you choose here.